cardinal/server/health.go

```diff
@@ -0,0 +1,21 @@
+package server
+
+import (
+	"github.com/go-openapi/runtime"
+	"github.com/go-openapi/runtime/middleware/untyped"
+	"pkg.world.dev/world-engine/cardinal/ecs"
+)
+
+func (handler *Handler) registerHealthHandlerSwagger(world *ecs.World, api *untyped.API) error {
```

Comment:
the `handler` receiver already has a ref to `World`, so can remove it from the args here